src/themes/palette.ts

```diff
@@ -1,67 +1,114 @@
-  backgroundSuccess: '#198754',
-  backgroundError: '#dc3545',
-  backgroundWarning: '#ffc107',
-  backgroundLight: '#f8f9fa',
-  backgroundDark: '#212529',
-  backgroundTransparent: '#transparent',
-  backgroundWhite: '#fff',
-  backgroundAccent: '#F6F7FB',
+//   // BUTTONS -background
 
-  // BUTTONS -background
+//   buttonPrimary: '#422ed1',
+//   buttonPrimaryDisabled: 'rgba(66, 46, 209, 0.45)',
+//   buttonSecondary: 'rgba(142, 144, 166, 1)',
+//   buttonSecondaryBorder: 'rgba(142, 144, 166, 1)',
+//   buttonSecondaryHover: '#fff',
+//   buttonSecondaryDisabled: 'rgba(66, 46, 209, 0.45)',
+//   buttonGhost: '#fff',
+//   buttonGhostHover: '#422ed1',
+//   buttonGhostText: '#8E90A6',
+//   buttonGhostBorder: '#8E90A6',
+//   ghostHoverText: '#422ed1',
 
-  buttonPrimary: '#422ed1',
-  buttonPrimaryDisabled: 'rgba(66, 46, 209, 0.45)',
-  buttonSecondary: 'rgba(142, 144, 166, 1)',
-  buttonSecondaryBorder: 'rgba(142, 144, 166, 1)',
-  buttonSecondaryHover: '#fff',
-  buttonSecondaryDisabled: 'rgba(66, 46, 209, 0.45)',
-  buttonGhost: '#fff',
-  buttonGhostHover: '#422ed1',
-  buttonGhostText: '#8E90A6',
-  buttonGhostBorder: '#8E90A6',
-  ghostHoverText: '#422ed1',
+//   buttonSuccess: '#0B7247',
+//   buttonSuccessActive: '#276738',
+//   buttonSuccessHover: '#367b48',
 
-  buttonSuccess: '#0B7247',
-  buttonSuccessActive: '#276738',
-  buttonSuccessHover: '#367b48',
+//   buttonWarning: '#CAB23F',
+//   buttonWarningActive: '#b49e35',
 
-  buttonWarning: '#CAB23F',
-  buttonWarningActive: '#b49e35',
+//   buttonWarningHover: '#cab23f',
+//   buttonError: '#DB2A30',
+//   buttonErrorActive: '#DB2A30',
+//   buttonErrorHover: '#b54248',
+// };
+// export type Palette = typeof palette;
 
-  buttonWarningHover: '#cab23f',
-  buttonError: '#DB2A30',
-  buttonErrorActive: '#DB2A30',
-  buttonErrorHover: '#b54248',
+export const palette = {
+  green: {
+    greenDefault: '#2ba745',
+    greenLight: '#0f5132', //green text color
+    greenLighter: '#d1e7dd', //alert bg
+    greenDark: '#065f43', //for icons
+  },
+  royalBlue: {
+    royalBlueDefault: '#422ED1', //button backgrounds
+    royalBlueDark: '#081336', //side menus
+    royalBlueLight: '#d9d5f6', //alert, icon backgrounds
+  },
+  blue: {
+    blueDefault: '#084298', //texts
+    blueLight: '#cfe2ff', //for bg
+    blueLighter: '#b6d4fe', //for border
+  },
+  red: {
+    redDefault: '#dc3545',
+    redLight: '#842029', //text color on alert bg
+    redLighter: '#f8d7da', // alert bg
+    redLightest: '#f5c2e7', //borders
+  },
+  yellow: {
+    yellowDefault: '#dc3545',
+    yellowLight: '#d3c789', //button bg
+    yellowLighter: '#664d03', //for text
+    yellowLightest: '#fff3cd',
+  },
+  grey: {
+    greyDefault: '#28293d', // default headings
+    greyLight: '#6B7280',
+    greyLighter: '#F0F0F0',
+    greyLightest: '#555770', //for placeholders
+    greyText: '#41464b', // text on grey bg
+    greyBackground: '#8E90A6', //secondary btn color
+  },
+  neutrals: {
```

Comment:
This introduces inconsistency. Only color names can appear on this object. Don't use terms like 'ghost'. It's not obvious what's the shade. Follow the above format. Keep the property names consistent.